docs/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+export const Footer = () => {
+  return (
+    <footer
+      className={`not-prose sticky top-0 z-50 
+        flex flex-wrap items-center justify-between
+        bg-white
+        py-5 px-4 shadow-md
+        shadow-slate-900/5 
+        dark:bg-slate-900/95 dark:shadow-none
+        sm:px-6 lg:px-8`}
```

Comment:
Instead of a single multiline string for these, I think I'd prefer an approach that Prettier can format so that the code formatted consistently across files and developers:

```suggestion
      className={clsx(
        'not-prose sticky top-0 z-50',
        'flex flex-wrap items-center justify-between',
        'bg-white',
        'py-5 px-4 shadow-md',
        'shadow-slate-900/5',
        'dark:bg-slate-900/95 dark:shadow-none',
        'sm:px-6 lg:px-8'
      )}
```

Reply:
I couldn't agree more. Thanks for the tip!